src/java.base/linux/classes/jdk/internal/platform/CgroupSubsystemFactory.java

```diff
@@ -220,6 +224,11 @@ public static Optional<CgroupTypeResult> determineType(String mountInfo,
      */
     private static void setCgroupV2Path(Map<String, CgroupInfo> infos,
                                         String[] tokens) {
+        String name = tokens[1];
+        if (!name.equals("")) {
+            // This is probably a v1 controller that we have ignored (e.g., freezer)
```

Comment:
Could we add an assertion that the controller isn't in the `infos` map?
```
           if (!name.equals("")) {
            // This must be a v1 controller that we have ignored (e.g., freezer)
            assert infos.get(name) == null;
            return;
         }
```

Reply:
Fixed.